src/redux/FilmSlice/FilmType.tsx

```diff
@@ -0,0 +1,18 @@
+export interface filmInformation {
```

Comment:
Film

Reply:
I changed the name filmInformation to Film